organisaatio-ui/src/ValidationSchemas/PerustietolomakeSchema.ts

```diff
@@ -1,34 +1,43 @@
 import Joi from 'joi';
+import JoiDate from '@joi/date';
```

Comment:
olisiko helpompi näin:
`import Joiless from 'joi';
import JoiDate from '@joi/date';
const Joi = Joiless.extend(JoiDate);`

Reply:
Fixd